### ISSUE TITLE
Enhance gulp generators

### ISSUE THREAD

**Issue by jmaicaaan** [jmaicaaan/express-starter-ts#19]

**Comment by ciiidi** [comment]
hmm I've been thinking if its possible that we can integrate this on cli tool

**Comment by jmaicaaan** [comment]
Yes that is correct. Generators, migrations, and seeds will all sit down into the cli. However, with that aren't we making the cli as a dependecy of our app too much? Would it be fine later on?
